packages/web-components/src/color/neutral-fill-card.ts

```diff
@@ -5,7 +5,7 @@ import { findClosestSwatchIndex, getSwatch } from './palette';
 const neutralCardFillAlgorithm: SwatchResolver = (designSystem: DesignSystem): Swatch => {
   const offset: number = neutralFillCardDelta(designSystem);
   const index: number = findClosestSwatchIndex(neutralPalette, backgroundColor(designSystem))(designSystem);
-  return getSwatch(index - (index < offset ? offset * -1 : offset), neutralPalette(designSystem));
```

Comment:
Ideally this update comes in from a rebase w/ master. Can you remove these in favor of just the color-vNext work? That'll keep everything conflict free.